src/styles/home.scss

```diff
@@ -281,6 +281,76 @@
   @include primary-cta;
 }
 
+// ---------------------------------------------------------------------------
+// Launch Sequences
+// ---------------------------------------------------------------------------
+
+
+  .launch__content {
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+    margin: 80px 0 40px 0;
+    width: 100%;
+  }
+  
+  
+  .launch__header {
+    @include h4-text;
+    color: $pale-yellow !important;
+  }
+  
+  .launch__countdown{
+    @media (min-width: $sm) {
+      font-size: 72px;
+    }
+    font-size: 40px;
+    margin-top: 0.32em;
+    margin-bottom: 0.04em;
+    line-height: 1.0;
+  }
+  
+  .launch__labels {
+    display: inline-block;
+    flex-basis: flex;
+  }
+
+  li {
+    @media (min-width: $sm) {
+      padding-left: 2.5em;
+      padding-right: 1.6em; 
+      font-size: 20px;
+    }
+    display: inline-block;
+    font-size: 16px;
+    padding-left: 1.5em;
+    padding-right: 0.3em;
+  }
+  
+  .launch__cta-container {
+    display: flex;
+    justify-content: center;
+    @media (min-width: $sm) {
+      margin-top: 2.4em;
+      display: block;
+    }
+    margin-bottom: 6.6em;
+  }
+
+  .launch-button {
+    color: #ffffff;
+    background-color: #4366e1;
+    font-size: 20px;
+    transition: background-color 0.3s ease;
+    padding: 12px 35px;
+    border-radius: 50px;
```

Comment:
I thought there was existing styling for CTA, maybe have jess take a look since she worked on that